charts/sn-platform/values.yaml

```diff
@@ -2194,6 +2194,8 @@ vault:
   serviceAccount:
     created: true
     name: ""
+  # time to live for service-account and user
+  ttl: 12h
```

Comment:
Perhaps we should consider using this configuration

```
oidcToken:
   ttl: 12h
```

 because it is for the vault oidc token

Reply:
good idea, thanks.

Reply:
fixed, PTAL, thanks.